ci_framework/roles/sosreport/defaults/main.yml

```diff
@@ -0,0 +1,259 @@
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_sosreport"
+cifmw_sosreport_nodes:
+  - 127.0.0.1
+cifmw_sosreport_nodes_sshkey: "~/.ssh/id_ecdsa"
+cifmw_sosreport_tmp_dir: "/tmp"
+cifmw_sosreport_enable_plugins:
+  - abrt
+  - acpid
+  - activemq
+  - ansible
+  - apache
+  - arcconf
+  - atomichost
+  - autofs
+  - azure
+  - bcache
+  - boom
+  - btrfs
+  - buildah
+  - candlepin
+  - ceph_ansible
+  - ceph_iscsi
+  - ceph_mds
+  - ceph_mgr
+  - ceph_mon
+  - ceph_osd
+  - ceph_rgw
+  - cifs
+  - clear_containers
+  - cloud_init
+  - cman
+  - cobbler
+  - collectd
+  - collectl
+  - composer
+  - conntrack
+  - container_log
+  - containerd
+  - convert2rhel
+  - corosync
+  - crio
+  - cs
+  - ctdb
+  - cups
+  - dellrac
+  - dhcp
+  - discovery
+  - distupgrade
+  - dlm
+  - dmraid
+  - docker
+  - docker_distribution
+  - dovecot
+  - drbd
+  - ds
+  - elastic
+  - etcd
+  - fapolicyd
+  - fcoe
+  - fibrechannel
+  - flatpak
+  - foreman
+  - foreman_installer
+  - foreman_openscap
+  - foreman_proxy
+  - frr
+  - fwupd
+  - gcp
+  - gdm
+  - gfs2
+  - gluster
+  - gluster_block
+  - grafana
+  - grub
+  - haproxy
+  - hpasm
+  - hpssm
+  - hyperv
+  - infiniband
+  - insights
+  - ipa
+  - ipmitool
+  - iprconfig
+  - ipvs
+  - iscsitarget
+  - java
+  - kata_containers
+  - keepalived
+  - kernelrt
+  - kimchi
+  - kubernetes
+  - leapp
+  - libreswan
+  - lightdm
+  - lilo
+  - lstopo
+  - lustre
+  - manageiq
+  - megacli
+  - memcached
+  - microshift
+  - microshift_ovn
+  - migration_results
+  - mongodb
+  - monit
+  - mpt
+  - mssql
+  - mvcli
+  - mysql
+  - named
+  - navicli
+  - nfsganesha
+  - nginx
+  - nodejs
+  - npm
+  - nscd
+  - nss
+  - ntp
+  - nvidia
+  - nvmetcli
+  - oddjob
+  - omnipath_client
+  - omnipath_manager
+  - omsa
+  - opencl
+  - opencontrail
+  - opendaylight
+  - opengl
+  - openshift
+  - openshift_ovn
+  - openstack_ansible
+  - openstack_aodh
+  - openstack_ceilometer
+  - openstack_cinder
+  - openstack_database
+  - openstack_designate
+  - openstack_glance
+  - openstack_heat
+  - openstack_horizon
+  - openstack_instack
+  - openstack_ironic
+  - openstack_keystone
+  - openstack_manila
+  - openstack_mistral
+  - openstack_neutron
+  - openstack_nova
+  - openstack_novajoin
+  - openstack_octavia
+  - openstack_placement
+  - openstack_sahara
+  - openstack_swift
+  - openstack_tripleo
+  - openstack_trove
```

Comment:
from this list, can we enable only openstack_nova, openstack_neutron for EDPM jobs. We can iterate this list based on the needed